.github/workflows/chartReleaser.yml

```diff
@@ -4,28 +4,34 @@ on:
   release:
     types:
       - published
+  pull_request:
+    paths:
+      - 'helm/**'
   workflow_dispatch:
     inputs:
       chart_version:
-        description: 'The chart version to create. Must be a valid semver string and should indicate that it is not a release chart version (eg. 5.1.2-pr36).'
+        description: 'The chart version to create. Will be appended to the current version from helm/Chart.yaml (eg. "pr36" on Chart.yaml version: 5.1.2 produces "5.1.2-pr36").'
```

Comment:
This isn't the most clearly worded (although the example is very useful). This is a sub-version or suffix essentially, right?

Reply:
Yeah reading it back it wasn't very clear. How does this one read?